roles/edpm_telemetry_power_monitoring/tasks/configure.yml

```diff
@@ -113,6 +113,14 @@
   when:
     - tls_crt_stat.stat.exists and tls_key_stat.stat.exists
   block:
+    - name: Create config file for exporters
+      ansible.builtin.include_tasks:
+        file: exporter_tls.yml
+      loop:
+        - ceilometer_prom_exporter
+      loop_control:
+        loop_var: exporter
+
```

Comment:
why a loop with 1 item?

Reply:
I'm adding that to follow the same technique in the ceilometer_compute config for any future exporters.